tools/wptrunner/wptrunner/browsers/firefox.py

```diff
@@ -752,6 +752,9 @@ def _set_required_prefs(self, profile):
 
         if self.test_type == "wdspec":
             profile.set_preferences({"remote.prefs.recommended": True})
+            profile.set_preferences({
+                "geo.provider.network.url": "https://web-platform.test:8444/webdriver/tests/support/http_handlers/geolocation_override.py"
```

Comment:
I wonder if this should be located in a more central place for wpt in general so it can also be used by other tests than webdriver.

Reply:
Maybe, but it can be always moved at later time. I wouldn't do it now to avoid breaking existing tests, especially when it's going to be downstreamed to mozilla-central.

Reply:
Filed: https://bugzilla.mozilla.org/show_bug.cgi?id=1960291